src/Draco.Compiler.Tests/Semantics/DocumentationCommentsTests.cs

```diff
@@ -221,7 +221,7 @@ public class TestClass { }
         // Act
         var compilation = Compilation.Create(
             syntaxTrees: ImmutableArray.Create(tree),
-            metadataReferences: Basic.Reference.Assemblies.Net70.ReferenceInfos.All
+            metadataReferences: Basic.Reference.Assemblies.Net80.ReferenceInfos.All
                 .Select(r => MetadataReference.FromPeStream(new MemoryStream(r.ImageBytes)))
```

Comment:
I feel like we are doing this so often that we could just have a constant in the test suite somewhere. Let's open an issue to clean this up

Reply:
Or even a property in test classes, so in the future we could parameterize which version we test with.